src/js/shot-generator/components/Three/InteractionManager.js

```diff
@@ -338,11 +339,11 @@ const InteractionManager = connect(
         if(dragTarget.target.userData.type === 'character') {
           let ikRig = SGIkHelper.getInstance().ragDoll;
           if(!ikRig || !ikRig.isEnabledIk && !ikRig.hipsMoving && !ikRig.hipsMouseDown) {
-            drag({ x, y }, dragTarget.target, camera, selections)
+            drag({ x, y }, dragTarget.target, camera, selections, isCtrlPressed.current)
```

Comment:
`PointerEvent` has a `ctrlKey` property, so you could check `event.ctrlKey` here in the `onPointerMove` handler (instead of adding the separate keyboard listeners and maintaining a ref).